### PR TITLE
Refresh family goal after approve/decline donations (KIDS-674)

### PR DESCRIPTION
**Pull request by MaikelStuivenberg**

### CODE REVIEW

File: lib/features/children/parental_approval/widgets/parental_approval_dialog_content.dart [hunk 1]
```diff
@@ -26,6 +26,9 @@ class ParentalApprovalDialogContent extends StatelessWidget {
   Future<void> _refreshHistory(BuildContext context) async {
     await context.read<FamilyOverviewCubit>().fetchFamilyProfiles();
     if (!context.mounted) return;
+    
+    await context.read<GoalTrackerCubit>().getGoal();
```
Comment (Daniela510) [inline lib/features/children/parental_approval/widgets/parental_approval_dialog_content.dart:30]: I would also remove it from the "parallel" below

File: lib/features/children/parental_approval/widgets/parental_approval_dialog_content.dart [hunk 2]
```diff
@@ -26,6 +26,9 @@ class ParentalApprovalDialogContent extends StatelessWidget {
...
+    if (!context.mounted) return;
 
     // Exectue the following cubits in parallel
     await Future.wait([
```
Comment (Daniela510) [inline lib/features/children/parental_approval/widgets/parental_approval_dialog_content.dart:34]: I would also remove getGoal from the "parallel" below, to not have it twice in this method

Comment (MaikelStuivenberg) [reply lib/features/children/parental_approval/widgets/parental_approval_dialog_content.dart:34]: Now you say this i'm actually wondering why that one doesnt work.. I'll do some more tests!

Comment (Daniela510) [reply lib/features/children/parental_approval/widgets/parental_approval_dialog_content.dart:34]: also why  is `if (!context.mounted) return;` twice also

Comment (MaikelStuivenberg) [reply lib/features/children/parental_approval/widgets/parental_approval_dialog_content.dart:34]: https://dart.dev/tools/linter-rules/use_build_context_synchronously :)

**Pull request closed without merging**